app/views/welcome/_works_list.html.haml

```diff
@@ -1,6 +1,5 @@
-- cache cache_key, expires_in: 24.hours do
```

Comment:
caching from _works_list partial is removed, now we cache popular and new works as a single fragment in _homepage partial